DiffAugment-stylegan2-pytorch/metrics/accuracy.py

```diff
@@ -45,6 +47,27 @@ def compute_accuracy(opts, batch_size=32):
         validation_accuracy = validation_correct / validation_all
     else:
         validation_accuracy = None
-    
 
     return train_accuracy, validation_accuracy
+
+
+def compute_accuracy_generated(opts, batch_size=32, diff_aug=False):
+    D = copy.deepcopy(opts.D).eval().requires_grad_(False).to(opts.device)
+    G = copy.deepcopy(opts.G).eval().requires_grad_(False).to(opts.device)
+
+    train_correct = 0
+    train_all = 0
+    if opts.validation_dataset_kwargs != {}:
+        all_z = torch.randn([len(opts.validation_dataset), G.z_dim], device=opts.device)
+    else:
+        all_z = torch.randn([10000, G.z_dim], device=opts.device)
```

Comment:
Could this magic contant be parameterized in args?

Reply:
It's a default value